src/stories/SideNavigation.stories.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useState } from 'react'
+import {
+  Avatar,
+  Button,
+  NavigationItem,
+  SideNavigation as SideNavigationItemComp,
+  Text
+} from '../components'
+import Block from '../components/SideNavigation/Block'
+import logo from '../assets/images/logo.svg'
+import logoClosed from '../assets/images/logo-closed.svg'
+import avatar from '../assets/images/avatar.jpg'
+import { NavigationItemTypes } from '../components/SideNavigation/NavigationItem/types'
+
+export default {
+  title: 'Side Navigation',
+  component: SideNavigationItemComp,
+  argTypes: {
+    showAction: {
+      options: [true, false]
+    }
+  }
+}
+
+const Template = (args): JSX.Element => {
+  const [open, setOpen] = useState<boolean>(true)
+
+  return (
+    <SideNavigationItemComp setOpen={setOpen} open={open} {...args}>
+      <main>
+        <NavigationItem
+          As={() => (
+            <a href="/https://ameriabank.am">
+              <img src={open ? logo : logoClosed} alt="Product Logo" />
+            </a>
+          )}
+          type={NavigationItemTypes.BLOCK_HEADER}
+          open={open}
+          showAction={args.showAction}
+          actionElm={args.actionElm}
+        />
+        <Block open={open} label="Finance">
+          <>
+            <NavigationItem
+              As={() => <a href="/https://ameriabank.am">Accounts</a>}
+              type={NavigationItemTypes.MAIN}
+              iconName="document"
+              open={open}
+              showAction={args.showAction}
+              actionElm={args.actionElm}
+              active
+            />
+            <NavigationItem
```

Comment:
according to comment below this will we
<NavigationItem
...>
<NavigationItem
                    As={() => <a href="/https://ameriabank.am">Accounts</a>}
                    type={NavigationItemTypes.MAIN}
                    iconName="document"
                    open={open}
                  />
                  <NavigationItem
                    As={() => <a href="/https://ameriabank.am">Savings</a>}
                    type={NavigationItemTypes.MAIN}
                    iconName="savings"
                    open={open}
                  />
</<NavigationItem>